lib/wahwah/tag.rb

```diff
@@ -31,6 +31,9 @@ def initialize(io, from_path: false)
       @file_size = io.size
       @file_io = io
 
+      # Rewinding at the start just in case some other helper class or parser already read from the file
+      @file_io.rewind
+
```

Comment:
@aidewoode Added `rewind` to the `Tag` class instead of the specific `RiffTag` class